scripts/usb_reset.py

```diff
@@ -90,6 +89,48 @@ def get_cg_dir(domid):
     return "/sys/fs/cgroup/devices/qemu-{}".format(domid)
 
 
+def get_ids_path(device):
+    usb_dir = "/var/run/nonpersistent/usb"
+    if not os.path.isdir(usb_dir):
+        os.makedirs(usb_dir)
```

Comment:
This is racy. It is probably not a problem but should we do something like

```python
import errno

try:
    os.makedirs(usb_dir)
except os.error as err:
   if err.errno != errno.EEXIST:
     raise
```

or is not worth it?

Reply:
Good spot, we should definitely do it this way.

Reply:
Done.

Reply:
actually I can also add this check at line 292, considering racing problem.